### PR TITLE
Message Logging

### PR DESCRIPTION
**Pull request by Mohit177**
This PR adds schema for message table and adds db writes and reads for the table.

### CODE REVIEW

File: whatsapp_fn.py [hunk 1]
```diff
@@ -54,6 +58,7 @@ def send_whatsapp_message(url,to,message,preview_url=False):
     return response
 
 def send_whatsapp_message_image(url,to,media_link,media_caption):
+    add_message(to, bot_number, to, media_link, "image", "whatsapp", "outgoing")
```
Comment (satishck1992) [inline whatsapp_fn.py:61]: Every message will have two copies. One for sender and one for receiver. Can you also add the second copy ?

File: whatsapp_fn.py [hunk 2]
```diff
@@ -102,6 +107,7 @@ def Get_Message():
         if records_a.shape[0]>0:
             hist = 'v'
             name = records_a.loc[0,'name']
+            add_message(frm, frm, bot_number, text, "text", "whatsapp", "incoming")
```
Comment (satishck1992) [inline whatsapp_fn.py:110]: Same as above @Mohit177

File: whatsapp_fn.py [hunk 3]
```diff
@@ -41,6 +44,7 @@ def get_auth_key():
 
 
 def send_whatsapp_message(url,to,message,preview_url=False):
+    add_message(to, bot_number, to, message, "text", "whatsapp", "outgoing")
```
Comment (satishck1992) [inline whatsapp_fn.py:47]: We don't need the `outgoing` tag since each the direction will be uniquely identified by `from` and `to`. What do you think ?

Comment (Mohit177) [reply whatsapp_fn.py:47]: Yes, this was a feature ask from @sidPhoenix17 as this tag will come handy to the person auditing the log.

File: whatsapp_fn.py (hunk 1 above)
Comment (Mohit177) [reply whatsapp_fn.py:61]: I can only see one set of message going in each conversation. Not able to see in the code where the copy is created on either side. Can you comment on the relevant section.

Comment (Mohit177) [reply whatsapp_fn.py:61]: @satishck1992 as discusses over call I am marking this comment as not relevant.

File: whatsapp_fn.py (hunk 2 above)
Comment (Mohit177) [reply whatsapp_fn.py:110]: @satishck1992 as discusses over call I am marking this comment as not relevant now.

File: whatsapp_fn.py (hunk 3 above)
Comment (Mohit177) [reply whatsapp_fn.py:47]: @satishck1992 as discusses over call I am marking this comment as not relevant now.

**Pull request merged**